backend/launch.ts

```diff
@@ -0,0 +1,316 @@
+  // Create base set of languages
+
+  await db.language.create({
+    data: {
+      name: 'Mandarin Chinese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Spanish',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'English',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Hindi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Bengali',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Portuguese',
+      dialect: 'European',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Portuguese',
+      dialect: 'Brazilian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Russian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Japanese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Western Punjabi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Marathi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Telugu',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Turkish',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Korean',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'French',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'German',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Vietnamese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Tamil',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Urdu',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Javanese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Italian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Arabic',
+      dialect: 'Egyptian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Gujarati',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Farsi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Hakka Chinese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Indonesian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Malay',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Ukranian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Romanian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Dutch',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Thai',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Greek',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Czech',
+    },
+  })
+
+  // Create topics
+  await db.topic.create({
+    data: {
+      name: 'Cooking',
+    },
+  })
+  await db.topic.create({
+    data: {
+      name: 'Rock Climbing',
```

Comment:
This made me think of a similar issue with variations. @nlitwin and I have talked before about "topics" and then "subtopics". 

At the end of the day, I'd love for us to figure out what the simplest implementation and system here is...

One idea is similar to my thought above: maybe we have `name: "Climbing"` and `variation: "Rock"` (😂). 

Or just how I have done it here with "Rock Climbing" and "Bouldering" and so on as separate topics.

These could essentially be used as "tags" where you add all that apply.

Reply:
I think allowing user entry is going to be important here, I don't think we're going to ever come up with an exhaustive list of human interests. We can do some lexical normalization, e.g. standardizing "Rock climbing", "rockclimbing" and "Rock-Climbing" but "Bouldering" is going to require some semantic understanding of the interest. I think having an "topic alias" table might make sense, so we have one canonical "Rock Climbing" row in topics, but we consult the alias table to find out "Boldering" is another name for the same thing. Topics can be merged and aliased as people create new topics to try and get as much activity on each topic as possible